packages/divider-block/src/settings.ts

```diff
@@ -89,8 +77,8 @@ export const settings: BlockSettings = {
                         {
```

Comment:
I would remove the surrounding `sectionHeading`. Imo it is clear that it is a `Line` when Line is selected and other blocks don't have it either.

Reply:
design is like this. Therefore, just keep it as it is